### DISCUSSION TITLE
ppstructure样例画不出README里半透明的彩色目标检测框

### DISCUSSION THREAD

**Discussion by charliedream1** [PaddlePaddle/PaddleOCR#12678] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu 20.04.5
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：2.7
- 运行指令/Command Code：
- 完整报错/Complete Error Message：无

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

========================================

问题：
README里的版面分析，是彩色半透明框。

<img width="422" alt="捕获" src="https://github.com/PaddlePaddle/PaddleOCR/assets/15007828/4c6dba01-9cc6-411a-aae7-ce2dff3199aa">

运行样例代码，画出的：

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/15007828/69688368-c3bf-4a0b-8f85-db00afe2d358)

查看接口也没有可以配置的，怎么画出README里的半透明目标检测框的风格？

=======================================
使用的代码如下：

```python
import os
import cv2
from paddleocr import PPStructure,draw_structure_result,save_structure_res

table_engine = PPStructure(show_log=True)

save_folder = './output'
img_path = 'ppstructure/docs/table/1.png'
img = cv2.imread(img_path)
result = table_engine(img)
save_structure_res(result, save_folder,os.path.basename(img_path).split('.')[0])

for line in result:
    line.pop('img')
    print(line)

from PIL import Image

font_path = 'doc/fonts/simfang.ttf' # PaddleOCR下提供字体包
image = Image.open(img_path).convert('RGB')
im_show = draw_structure_result(image, result,font_path=font_path)
im_show = Image.fromarray(im_show)
im_show.save('result.jpg')
```

**Comment by tink2123** [comment, marked as answer]
参考PP-OCR里的画图代码：https://github.com/PaddlePaddle/PaddleOCR/blob/52cf1e1bdd2a4422bd6232e924bf26b85a47afda/tools/infer/utility.py#L428
